river/anomaly/base_transformer.py

```diff
@@ -0,0 +1,85 @@
+from abc import ABC, abstractmethod
+
+#import abc
+import numpy as np
+import math
+import utils
+
+#from pysad.utils import _iterate
+
+
+class BaseTransformer(ABC):
```

Comment:
What does this class do differently than the existing `Transformer` class in the `base` module? Do you really need to introduce a new base class?